package.json

```diff
@@ -50,9 +50,9 @@
   },
   "scripts": {
     "jshint": "jshint -c .jshintrc lib test",
-    "test-pouchdb": "cd node_modules/pouchdb-server && npm run test-pouchdb",
-    "test-pouchdb-minimum": "cd node_modules/pouchdb && SERVER=express-pouchdb-minimum npm test",
-    "test-couchdb": "cd node_modules/pouchdb-server && npm run test-couchdb",
+    "test-pouchdb": "./bin/test-setup.sh && ./bin/test-pouchdb.sh",
+    "test-pouchdb-minimum": "./bin/test-setup.sh && cd node_modules/pouchdb && SERVER=express-pouchdb-minimum npm test",
```

Comment:
^ this is using `node_modules/pouchdb` instead of `node_modules/pouchdb-server`, so it's actually still not testing anything. My bad.